src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir , serif;
+  font-weight: 300;
+}
+
+/* #region form */
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+/* #endregion */
```

Comment:
good that you try to introduce structure, but when there's just one style - it's self-explanatory 🙂